### PR TITLE
sysbuild: make SECURE_BOOT Kconfig selection automatic

### PR DESCRIPTION
**Pull request by nordicjm**

### CODE REVIEW

File: sysbuild/Kconfig.secureboot [hunk 1]
```diff
@@ -2,25 +2,29 @@
...
-	  Set this option to enable the first stage bootloader which
-	  verifies the signature of the app.
-
-if SECURE_BOOT
+	  This option will be set if the first stage bootloader which
+	  verifies the signature of the app is enabled
 
 config SECURE_BOOT_APPCORE
 	bool "Appcore"
+	select SECURE_BOOT
```
[the remark this answers is not in the document]
Comment (SebastianBoe) [reply sysbuild/Kconfig.secureboot:15]: "Just the fact that both SECURE_BOOT_APPCORE and SECURE_BOOT_NETCORE can be enabled at same time seems wrong."

Having 2 bootloaders, one for each core is a valid use-case FYI.

[the remark this answers is not in the document]
Comment (nordicjm) [reply sysbuild/Kconfig.secureboot:15]: > Just the fact that both SECURE_BOOT_APPCORE and SECURE_BOOT_NETCORE can be enabled at same time seems wrong. Should a choice have been used instead ?

Nothing wrong with it, intended behaviour

[the remark this answers is not in the document]
Comment (tejlmand) [reply sysbuild/Kconfig.secureboot:15]: as well as the first setting:
> config SECURE_BOOT
> ...
>	help
>	  This option will be set if the first stage bootloader which
>	  verifies the signature of the app is enabled

all settings talks only about a single first stage bootloader.

So text should reflect that there can be multiple secure bootloaders.

[the remark this answers is not in the document]
Comment (nordicjm) [reply sysbuild/Kconfig.secureboot:15]: Let's defer this to another PR

**Comment by nordicjm** [comment]
@hakonfam @sigvartmh @oyvindronningstad @milewr @frkv @Vge0rge @vili-nordic @mswarowsky @Damian-Nordic @kkasperczyk-no @MarekPieta @carlescufi @tejlmand please review

**Review by Damian-Nordic — APPROVED** [review]

**Review by anangl — APPROVED** [review]

**Pull request merged**